dll/apisets/api-ms-win-core-localization-l1-1-0.spec

```diff
@@ -5,7 +5,7 @@
 @ stdcall -version=0x601+ FindNLSString() kernel32.FindNLSString
 @ stdcall -version=0x601+ FindNLSStringEx() kernel32.FindNLSStringEx
 @ stdcall -version=0x601+ GetACP() kernel32.GetACP
-@ stdcall -version=0x601+ GetCPFileNameFromRegistry() kernel32.GetCPFileNameFromRegistry
+@ stdcall -version=0x501-0x600 GetCPFileNameFromRegistry() kernel32.GetCPFileNameFromRegistry
```

Comment:
Unwanted change?

Reply:
This was intentional as well; see my commit comment on a4713a054901971bdc85045b28efb38d6fe68276 for detaials.

Reply:
Whatever the error was, that is a wrong fix, as apisets are all NT6+ at least.